### PR TITLE
Feature/dsm up dn counter

### PR DESCRIPTION
**Pull request by georgiosantoniadis**

### CODE REVIEW

File: SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift [hunk 1]
```diff
@@ -0,0 +1,191 @@
+//
+//  Copyright © 2023 SBB. All rights reserved.
+//
+
+import SwiftUI
+
+public struct SBBUpDnCounterView: View {
```
Comment (fvessaz) [inline SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift:7]: For what stand this name? I do not get the "Up Dn" counter abbreviation...

Comment (georgiosantoniadis) [reply SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift:22]: This is the DSM Name for Stepper: https://www.figma.com/file/WOtLIam1xwrqcgnAITsEhV/Design-System-Mobile?type=design&node-id=372%3A11403&t=xNqeiEa4K00eOoTv-1

File: SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift
[the remark this answers is not in the document]
Comment (georgiosantoniadis) [reply SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift:1]: added

[the remark this answers is not in the document]
Comment (georgiosantoniadis) [reply SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift:1]: (also added Snapshot tests)

**Review by fvessaz — APPROVED** [review]

**Pull request merged**